scripts/s3manage.py

```diff
@@ -0,0 +1,415 @@
+#!/usr/bin/env python
```

Comment:
Yes, it's why it's `/usr/bin/env python`and not `/usr/bin/python`for instance. No ?